Sources/Purchasing/Purchases/Purchases.swift

```diff
@@ -1229,6 +1229,13 @@ public extension Purchases {
 #if os(iOS)
     /**
      * Displays a sheet that enables users to redeem subscription offer codes that you generated in App Store Connect.
+     *
+     * - Warning: Even though the docs in `SKPaymentQueue.presentCodeRedemptionSheet`
+     * say that it's available on Catalyst 14.0, there is a note:
+     *
+     * `This function doesn’t affect Mac apps built with Mac Catalyst.`
+     *
+     * when, in fact, it crashes when called both from Catalyst and also when running as "Designed for iPad".
```

Comment:
This is broken when rendering:
<img width="885" alt="image" src="https://user-images.githubusercontent.com/3922667/184418864-e05a0913-f56f-47a7-aa4b-11c50a393cac.png">

A couple of suggestions: 
- Using `important` instead of warning since the app won't build in catalyst because we're explicitly making it unavailable to protect from this runtime crash
- packing lines together so they become a part of the same block
- Adding quotes so it looks a little clearer

```suggestion
     /**
 * Displays a sheet that enables users to redeem subscription offer codes that you generated in App Store Connect.
 *
 * - Important: Even though the docs in `SKPaymentQueue.presentCodeRedemptionSheet`
 * say that it's available on Catalyst 14.0, there is a note:
 * "`This function doesn’t affect Mac apps built with Mac Catalyst.`"
 * when, in fact, it crashes when called both from Catalyst and also when running as "Designed for iPad".
 * This is why RevenueCat's SDK makes it unavailable in mac catalyst.
 */
```

Reply:
The changes would look like this

<img width="824" alt="image" src="https://user-images.githubusercontent.com/3922667/184419193-45e1c93f-29f1-40b2-beab-534380c2aaed.png">

Reply:
You can test this yourself by building the framework, then going to the generated docs in xcode with cmd + shift + 0

Reply:
alternatively `bundle exec fastlane preview_docs`

Reply:
This is what happens when I try to put the note within the note on a separate line to distinguish it from the rest of the original text.